pytorch_lightning/metrics/__init__.py

```diff
@@ -37,3 +39,8 @@
     R2Score,
     SSIM,
 )
+
+warn(
+    "`pytorch_lightning.metrics.*` module has been renamed to `torchmetrics.*` and split off to its own package (https://github.com/PyTorchLightning/metrics) since v1.3 and will be removed in v1.5",
```

Comment:
code checker is failing due to the line being too long
```suggestion
    "`pytorch_lightning.metrics.*` module has been renamed to `torchmetrics.*`"
    " and split off to its own package (https://github.com/PyTorchLightning/metrics)"
    " since v1.3 and will be removed in v1.5",
```